pkgs/by-name/da/dashy-ui/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  nodejs,
+  yq-go,
+  settings ? { },
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dashy-ui";
```

Comment:
This note can be removed once a full release comes, but I think it should stay as is for the current PR; the update cycle is somewhat slow at the moment.